recipes/lastversion/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "lastversion" %}
+{% set version = "1.5.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lastversion-{{ version }}.tar.gz
+  sha256: 8be68e865cfb70c8089ea9cb25803e0d8fe60435cc663b9e014858885b946214
+
+build:
+  number: 0
```

For easier maintenance, make this package `noarch`.
```suggestion
  number: 0
  noarch: python
```